packages/semi-icons/src/svgs/meta.json

```diff
@@ -1071,6 +1071,10 @@
         "name": "case_sensitive",
         "category": "Writing"
     },
+    {
+        "name": "fast_forward",
+        "category": "Music"
+    },
     {
```

Comment:
Please delete wrong spell icon in meta.json, so that will not show on site

Reply:
> Please delete wrong spell icon in meta.json, so that will not show on site
Thanks for review.I have modified it.Please take a look.